opendse-io/src/main/java/net/sf/opendse/io/ReaderTGFF.java

```diff
@@ -0,0 +1,575 @@
+package net.sf.opendse.io;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import net.sf.opendse.model.Application;
+import net.sf.opendse.model.Communication;
+import net.sf.opendse.model.Dependency;
+import net.sf.opendse.model.Link;
+import net.sf.opendse.model.LinkTypes;
+import net.sf.opendse.model.Mapping;
+import net.sf.opendse.model.Mappings;
+import net.sf.opendse.model.Resource;
+import net.sf.opendse.model.ResourceTypes;
+import net.sf.opendse.model.SpecificationTypeBased;
+import net.sf.opendse.model.Task;
+
+/**
+ * The {@code ReaderTGFF} imports an {@code Application}, {@code Mappings} and a

```

Comment:
Use `{@link Class}` everywhere. `@code` is only used for outlining source code, e.g.,` {@code null}`